### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by tortuga-feliz**
Hello!

Thank you for this very useful repo with exercises!  I learned a lot using it!
I think I found a small typo in the last exercise. Please check if my fix is correct 😸

Best regards,
Kasia

### CODE REVIEW

File: a.core_concepts.md [hunk 1]
```diff
@@ -290,7 +290,7 @@ kubectl describe nginx | grep val1
 
 # or
 
-kubectl run po busybox --restart=Never --image=busybox --env=var1=val1 -it --rm -- env
+kubectl run nginx --restart=Never --image=busybox --env=var1=val1 -it --rm -- env
```
Comment (dgkanatsios) [inline a.core_concepts.md]: this should be `kubectl run busybox`, since the image is 'busybox', no?

**Comment by dgkanatsios** [comment]
thanks for your kind words! Just a minor change and I'll be happy to accept your PR :)

File: a.core_concepts.md (hunk 1 above)
Comment (tortuga-feliz) [reply a.core_concepts.md]: Actually in the question says `nginx`, so I should fix the image..
image
Thanks for spotting 👁

Comment (dgkanatsios) [reply a.core_concepts.md]: oh, yes, you're absolutely right! thanks!

**Comment by tortuga-feliz** [comment]
@dgkanatsios done ✅ please have a look now 😸

**Comment by dgkanatsios** [comment]
thank you!

**Pull request merged**